### ISSUE TITLE
Handle simple error

### ISSUE THREAD

**Issue by xoac** [lovasoa/custom_error#16]
Let's say I wanna create simple Error like this `pub struct WrongValue(u32)` or `pub struct WrongValue`
The perfect way would be:
```rust
custom_error! {WrongValue(u32) = "Wrong value {0}"}
```
or
```rust
custom_error! {WrongValue = "Wrong value"}
```

Now I need to specify this like this:
```rust
custom_error!{ Wrong 
   Value { value: u32 } = "Wrong value {value}", 
}
```
Your macro is great to merge many errors together it would be nice to have another macro to create simpler Error (without source).

Let me know what do u think. I am new to macros but I could try create PR for that if you are interested.

**Comment by lovasoa** [comment]
This would be a nice feature indeed ! Pull requests are welcome :)

**Comment by lovasoa** [comment]
Fixed by #17

**Comment by lovasoa** [comment]
The new version with error structs was published as 1.4.0: https://crates.io/crates/custom_error/1.4.0
